File: src/main/java/ru/urfu/cake/shop/identity/service/dto/LoginRequestDto.java
package ru.urfu.cake.shop.identity.service.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class LoginRequestDto {

    @NotBlank(message = "email обязателен")
    @Email(message = "некорректный формат email")
    private String email;

    @NotBlank(message = "password обязателен")
    private String password;
}

File: src/main/java/ru/urfu/cake/shop/identity/service/dto/LoginRequestDto.kt
package ru.urfu.cake.shop.identity.service.dto

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

data class LoginRequestDto(
    @field:NotBlank(message = "email обязателен")
    @field:Email(message = "некорректный формат email")
    val email: String = "",

    @field:NotBlank(message = "password обязателен")
    val password: String = ""
)